src/ui/CardActionBar/CardActionBar.ts

```diff
@@ -0,0 +1,82 @@
+import {Component} from '../Base/Component';
+import {ComponentOptions} from '../Base/ComponentOptions';
+import {IComponentBindings} from '../Base/ComponentBindings';
+import {Initialization} from '../Base/Initialization';
+import {IQueryResult} from '../../rest/QueryResult';
+import {Assert} from '../../misc/Assert';
+import {$$} from '../../utils/Dom';
+
+export interface ICardActionBarOptions {
+  hidden?: boolean;
+}
+
+/**
+ * This component displays an action bar at the bottom of a Card result (see
```

Comment:
Same comment as the Backdrop component. The description is too short, and not of much use from an end user point of view.

How can I use it, exactly ? The component that can be placed inside, do they need anything special to work correctly ?

What happen if I place this outside of a Card template ? etc.